app/src/main/java/org/sil/gatherwords/room/SessionDao.java

```diff
@@ -25,6 +26,10 @@
 	@Query("SELECT :columns FROM session WHERE :comparisonString")
 	List<String> getWhere(List<String> columns, String comparisonString);
 
+	@Query("UPDATE session SET deletedAt = NULL WHERE deletedAt = " +
+			"(SELECT MAX(deletedAt) FROM session)")
+	void undoLastDeleted();
```

Comment:
This is an update, so I would move it down to ~line 44.

Reply:
Donezo